rpc/client_test.go

```diff
@@ -36,13 +36,14 @@ func init() {
 func TestClientHeartbeat(t *testing.T) {
 	addr := util.CreateTestAddr("tcp")
 
+	// Heartbeats are node-to-node requests: use server context for both server and client.
```

Comment:
Maybe server context should be named node context instead, since it can be used as a client too.